app/src/main/java/com/example/flexioffice/presentation/BookingViewModel.kt

```
@@ -61,10 +62,40 @@ class BookingViewModel
+            private const val COMMENT_KEY = "booking_comment"
+        }
+
         private val _uiState =
             MutableStateFlow(
-                BookingUiState(),
+                BookingUiState(
+                    selectedDate =
+                        savedStateHandle.get<String>(SELECTED_DATE_KEY)?.let {
+                            try {
+                                LocalDate.parse(it)
+                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)